statics/js/components/graph-layout.js

```diff
@@ -1154,7 +1154,12 @@ TopologyGraphLayout.prototype = {
     var totalByte = (metrics.RxBytes || 0) + (metrics.TxBytes || 0);
     var deltaMillis = metrics.Last - metrics.Start;
 
-    const fadeoutFactor = 2;
+    // FIXME: calculation assumes browser clock (new Data()) is in
+    // sync with agent clock (new Date(metrics.Last)) which incorrect
+    // in the general case:
+    // (1) different timezones (no currently addressed)
```

Comment:
for this last point we are using UTC time on the agent side, so I guess we should use UTC here too